crates/ruff_python_parser/src/parser/statement.rs

```diff
@@ -3242,6 +3242,7 @@ impl<'src> Parser<'src> {
                 None
             };
 
+            let default_start = self.node_start();
```

Comment:
nit: I think it's now the start of the `=` token instead so like `equal_token_start`?